modules/templates/ref-creating-templates.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * admin/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ref-creating-templates_{context}"]
+= Creating templates
+
+You can create a template by defining a `Template` object as a YAML file.
+
+The `Template` object describes the template and its metadata. It also contains required input variables and a list of actions which are executed by the scaffolding service.
+
+.`Template` object example
+[source,yaml]
+----
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: template-name # <1>
+  title: Example template # <2>
+  description: An example template for v1beta3 scaffolder. # <3>
+spec:
+  owner: backstage/techdocs-core # <4>
+  type: service # <5>
+  parameters: # <6>
+      # First set of steps
+    - title: Fill in some steps
+      required:
+        - name
+      properties:
+        name:
+          title: Name
+          type: string
+          description: Unique name of the component
+        owner:
+          title: Owner
+          type: string
+          description: Owner of the component
+    # Second set of steps
+    - title: Choose a location
+      required:
+        - repoUrl
+      properties:
+        repoUrl:
+          title: Repository Location
+          type: string
+  steps: # <7>
+    - id: fetch-base
+      name: Fetch Base
+      action: fetch:template
+      # ...
+  output: # <8>
+    links:
+      - title: Repository # <9>
```

Comment:
You need more than this to create a link to the code repository generate.  It would look something like:
```
- title: Repository
  url: ${{ steps['publish'].output.remoteUrl }}
```